packages/libs/eda/src/lib/map/analysis/mapTypes/plugins/DonutMarkerMapType.tsx

```diff
@@ -453,9 +485,75 @@ function MapOverlayComponent(props: MapTypeMapLayerProps) {
   const toggleStarredVariable = useToggleStarredVariable(props.analysisState);
   const noDataError = getLegendErrorMessage(data.error);
 
+  const updateLegendPosition = useDebouncedCallback(
+    (position: PanelConfig['position']) => {
+      setLegendPanelConfig({
+        ...legendPanelConfig,
+        pie: {
+          variable: position,
+        },
+      });
+    },
+    250
```

Comment:
Is the debouncing really needed? I found that the callback really is only called on drag completion. And we don't debounce the viz position and size below.